src/components/wizard/WizardStates.ts

```diff
@@ -4,16 +4,27 @@ const exclamationSmall = require('./assets/exclamationSmall.png');
 import {WizardStepsConfig} from './types';
 
 export const StatesConfig: WizardStepsConfig = {
-  enabled: {color: Colors.grey30, circleColor: Colors.grey60, enabled: true},
-  disabled: {color: Colors.grey50, circleColor: Colors.grey60},
-  error: {color: Colors.red30, icon: exclamationSmall, enabled: true, accessibilityInfo: 'Validation Error'},
-  skipped: {color: Colors.red30, enabled: true, accessibilityInfo: 'Not completed'},
+  enabled: {color: Colors.$textNeutralHeavy, circleColor: Colors.$outlineDisabled, enabled: true},
+  disabled: {color: Colors.$textDisabled, circleColor: Colors.outlineDefault},
+  error: {
+    color: Colors.$iconDangerLight,
+    circleColor: Colors.$outlineDanger,
+    icon: exclamationSmall,
+    enabled: true,
+    accessibilityInfo: 'Validation Error'
+  },
+  skipped: {
+    color: Colors.$textDangerLight,
+    circleColor: Colors.$outlineDanger,
+    enabled: true,
+    accessibilityInfo: 'Not completed'
+  },
   completed: {
-    color: Colors.grey30,
-    circleColor: Colors.grey60,
+    color: Colors.$iconNeutral,
+    circleColor: Colors.$outlineDisabled,
```

Comment:
```suggestion
    circleColor: Colors.$outlineDefault,
```

Reply:
Was it changed to grey50 so you're using the $outlineDisabled token?

Reply:
It's set to $outlineNeutralMedium which is $outlineDisabled